.eslintrc.cjs

```diff
@@ -64,12 +68,51 @@ module.exports = {
             allowNamedExports: true,
           },
         ],
-        "@typescript-eslint/no-unnecessary-type-assertion": "error",
```

Comment:
Why was this rule turned off?

Reply:
This line would be redundant because it's now enabled in the _`recommended-type-checked`_ config 😄 https://v6--typescript-eslint.netlify.app/rules/no-unnecessary-type-assertion